index.js

```diff
@@ -1,194 +1,289 @@
-const Checker = require('password-checker')
-const defaultMellt = require('./src/mellt.js')
+const R = require('ramda')
+
+const trees = require('./src/trees.mod.js')
+const words = require('./src/norvig-10000.mod.js')
+const names = require('./src/all-names.mod.js')
+const passwords = require('./src/passwords-10000.mod.js')
+const mellt = require('./src/mellt.js')
```

Comment:
To že jsem měl v tom ukázkovém kódu předaný ten mellt před nepovinný parametr v konstruktoru sloužilo k tomu, že by jsi někde mohl chtit vymenit tu implementaci melltu(treba v unit testech) takhle je to skryta zavislost. Pokud ti to nevadi tak je to samozrejme ok, obecne se v JS moc DI jeste nepoužívá.

Reply:
Stejne to musim implementovat jako nasi knihovnu protoze delame zasahy do kodu tak ze je to asi takto lepsi zaclenit to jako nas kod

Reply:
jj to říkám jako hint, samozřejme to tak klidně nechej, jen chci, aby jsi si uvedomoval co z toho plyne za důsledky :-D